lib/tasks/scsb.rake

```diff
@@ -9,4 +11,46 @@ namespace :scsb do
     %x[bundle exec loops stop 2>&1]
   end
 
+  desc "Index SCSB records with all changed records since SET_DATE, against SET_URL"
+  task updates: :environment do
```

Comment:
these are pretty long for rake tasks. what do you think about putting the logic in code, so it can be tested, and then calling it from the task?

Reply:
Makes sense. Easier to test.